server/fleet/calendar.go

```diff
@@ -41,6 +41,16 @@ type UserCalendar interface {
 	Get(event *CalendarEvent, key string) (interface{}, error)
 }
 
+// Lock interface for managing distributed locks.
+type Lock interface {
+	AcquireLock(ctx context.Context, name string, value string, expireMs uint64) (result string, err error)
```

Comment:
Please document what's `name`, what's `value`, what's `key` in each method.

Reply:
Documented.